spec/cohort.spec.js

```diff
@@ -0,0 +1,101 @@
+const Cohort = require('../src/cohort.js')
+const Student = require('../src/student.js')
+
+describe('Cohort', () => {
+  let cohort5
+  let arisaSigrist
+  let bobRoss
+  let michelleObama
+
+  beforeEach(() => {
```

Comment:
beforeEach makes the .spec file a lot shorter and avoids repetition. Need to implement on my repo